### DISCUSSION TITLE
SolidSphere coordinates to 3D array of points

### DISCUSSION THREAD

**Discussion by tom-new** [pyvista/pyvista#5666] [Q&A]
`pyvista.SolidSphere` returns an instance of `pyvista.UnstructuredGrid`, which makes sense as the grid nodes are not evenly spaced in Cartesian coordinates, and there may need to be connectivity between the first and last meridians. However, in spherical coordinates the grid nodes are spaced evenly, so in some sense it is a "structured" grid (aside from meridian connectivity). Is there a sensible way to recover this structure (for example, by reshaping `SolidSphere.get_array('mydata')` from (N,) to (i,j,k))?

**Comment by MatthewFlamm** [comment]
See https://github.com/pyvista/pyvista/discussions/4914#discussioncomment-7037610

**Reply by MatthewFlamm** [reply]
While the grid is structured using that methodology, you end up with degenerate points.  For example a lot of points at the origin. This may not be an issue for your use case.
